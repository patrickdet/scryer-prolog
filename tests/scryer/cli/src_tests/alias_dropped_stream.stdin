alias_dropped_stream.
halt.
