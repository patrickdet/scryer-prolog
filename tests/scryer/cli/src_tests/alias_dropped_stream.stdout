   error(existence_error(stream,readme),stream_property/0).
